### DISCUSSION TITLE
How to compare two commits using octokit api?

### DISCUSSION THREAD

**Discussion by rahiparikh** [octokit/octokit.js#2014] [Q&A]
Hello!

I am trying to compare two git commits using GH's compare API on push event. In the event, the base hash is `000000` and head is some other value. When I try to compare head SHA against `000000`, I get an error.

What is a recommended way of comparing one commit against start of the branch?

Thanks!

**Comment by gr2m** [comment]
I remember I've run into this before myself. But it's been a while, I don't know how I worked around it. As this is a question about GitHub's APIs, a better place to ask would be https://github.community/c/github-ecosystem/37, you are more likely to get an answer there, several hubbers are watching that forum. Please share if you find out the answer

**Reply by rahiparikh** [reply]
Thank you @gr2m! I have asked the question there. Hopefully, moderators will approve it.
